service/history/historyEngine2_test.go

```diff
@@ -848,6 +848,7 @@ func (s *engine2Suite) createExecutionStartedState(we types.WorkflowExecution, t
 }
 
 //nolint:unused
+//lint:ignore U1000 for printing within tests
```

Comment:
huh.  just because of the name?  maybe better to just change the name...

Reply:
This is currently not used anywhere, so we could remove it entirely.
Since I saw previous `nolint` I assumed this may be sometimes used for debugging in tests, so added additional declaration for staticcheck.
Not sure whether it is worth keeping such helpers though...